src/components/logs/toolbar.tsx

```diff
@@ -83,31 +92,54 @@ export function LogToolbar({
 					</div>
 
 					{/* Right side - Action Buttons */}
-					<div className="flex items-center gap-2">
-						<Button
-							size="md"
-							emphasis="subtle"
-							intent="secondary"
-							onClick={onCopyAll}
-							title="Copy all displayed logs"
-							className="bg-theme-surface-primary"
-						>
-							<Copy className="mr-1 h-4 w-4 fill-theme-text-secondary" />
-							Copy All
-						</Button>
+					<TooltipProvider>
+						<div className="flex items-center gap-2">
+							<Tooltip delayDuration={200}>
+								<TooltipTrigger>
+									<Button
+										size="md"
+										emphasis="subtle"
+										intent="secondary"
+										onClick={onReload}
+										className="bg-theme-surface-primary"
+									>
+										<Refresh className="h-4 w-4 fill-theme-text-secondary" />
+									</Button>
+								</TooltipTrigger>
+								<TooltipContent>Reload logs</TooltipContent>
+							</Tooltip>
 
-						<Button
-							size="md"
-							emphasis="subtle"
-							intent="secondary"
-							onClick={onDownload}
-							title="Download logs as file"
-							className="bg-theme-surface-primary"
-						>
-							<Download className="mr-1 h-5 w-5 fill-theme-text-tertiary" />
-							Download
-						</Button>
-					</div>
+							<Tooltip delayDuration={200}>
```

Comment:
Can this tooltip button be abstracted to a component, as the code for it is duplicated here 3 times.

Please make sure that the buttons are actually squares according to the design